### PR TITLE
G46 validate views

### PR DESCRIPTION
**Pull request by p013570**

### CODE REVIEW

**Comment by p013570** [comment]
It would be better if users don't have to specify element group properties and their classes. They should only need to specify any transient properties required, the other properties should be looked up in the schema.

**Comment by p013570** [comment]
We need to consider changing the view so you only have to specify transient properties. Standard properties should be read from the schema.

File: gaffer-core/store/src/main/java/gaffer/store/schema/Schema.java [hunk 1]
```diff
@@ -246,6 +289,15 @@ protected void addEntity(final String group, final SchemaEntityDefinition elemen
         super.addEntity(group, elementDef);
     }
 
+    @Override
+    public String toString() {
+        try {
+            return "Schema" + new String(toJson(true), "UTF-8");
```
Comment (a09631) [inline gaffer-core/store/src/main/java/gaffer/store/schema/Schema.java]: If encoding is a project level constant, this needs to be extracted and made available project-wide.

File: gaffer-core/store/src/test/java/gaffer/store/schema/SchemaElementDefinitionValidatorTest.java [hunk 2]
```diff
@@ -27,24 +27,192 @@
...
+        // Then
+        assertTrue(isValid);
+    }
 
     @Test
     public void shouldValidateAndReturnTrueWhenNoPropertiesAggregated() {
         // Given
         final SchemaElementDefinition elementDef = mock(SchemaElementDefinition.class);
         final SchemaElementDefinitionValidator validator = new SchemaElementDefinitionValidator();
+        ;
```
Comment (a09631) [inline gaffer-core/store/src/test/java/gaffer/store/schema/SchemaElementDefinitionValidatorTest.java]: Remove if unused.

**Pull request merged**